modules/nf-core/tabix/bgziptabix/main.nf

```diff
@@ -5,13 +5,14 @@ process TABIX_BGZIPTABIX {
     conda "bioconda::tabix=1.11"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
         'https://depot.galaxyproject.org/singularity/tabix:1.11--hdfd78af_0' :
-        'biocontainers/tabix:1.11--hdfd78af_0' }"
+        'quay.io/biocontainers/tabix:1.11--hdfd78af_0' }"
```

Comment:
You don't have to put the `quay.io` here anymore (this is now the default repo in the nf-core pipelines)